lib/mixlib/install/generator/powershell/scripts/helpers.ps1.erb

```diff
@@ -82,10 +82,14 @@ function Get-WebContentOnFullNet {
   
 
   if($bypassList -ne $null){
-     
-	 $bypassList = $bypassList.split(",")
-	 $proxy.BypassList = $byPassList
-  }
+  	$bypassList = $bypassList.split(",")
+    ForEach ($bypassListItem in $bypassList){
+        if ($bypassListItem -match '\*\..*'){
+          Write-Warning "Wildcard entry $bypassListItem will be ignored, please use regex"
```

Comment:
In some cases a `*` followed by `.` is valid. perhaps just test if the item validates to a propper regex string:

```
try {
  "test" -match $bypassListItem
} catch {
  Write-Warning "$bypassListItem will be ignored, please use a valid regular expression"
}

Reply:
Have created and tested a proper test using [regex] and added a continue so that the invalid entry is skipped.